lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosResult.cxx

```diff
@@ -104,11 +104,145 @@ String FunctionalChaosResult::__repr__() const
+      << "- inverse transformation=" << inverseTransformation_.getInputDimension() << " -> " << inverseTransformation_.getOutputDimension() << "\n"
+      << "- orthogonal basis dimension=" << orthogonalBasis_.getMeasure().getDimension() << "\n"
+      << "- indices size=" << indicesSize << "\n"
+      << "- relative errors=" << relativeErrors_ << "\n"
+      << "- residuals=" << residuals_ << "\n";
+  oss << "\n";
+
+  const Scalar ell_threshold = ResourceMap::GetAsUnsignedInteger("FunctionalChaosResult-PrintEllipsisThreshold");
+  const UnsignedInteger ell_size = ResourceMap::GetAsUnsignedInteger("FunctionalChaosResult-PrintEllipsisSize");
+  const UnsignedInteger columnWidth = ResourceMap::GetAsUnsignedInteger("FunctionalChaosResult-PrintColumnWidth");
+  const UnsignedInteger ellipsis = (indicesSize * outputDimension > ell_threshold);
+  const bool isEllipsisEnabled = (ellipsis and outputDimension > 2 * ell_size);
+  // Print table header
+  oss << "| Index |" << padString(" Multi-index ") << "|";
+  String intermediateString;
+  if (isEllipsisEnabled)
+  {
+    for (UnsignedInteger j = 0 ; j < ell_size; ++ j)
+    {
+      intermediateString = OSS() << " Coeff.#" << j;
+      oss << padString(intermediateString) << "|";
+    }
+    intermediateString = OSS() << " ... ";
+    oss << padString(intermediateString) << "|";
+    for (UnsignedInteger j = outputDimension - ell_size ; j < outputDimension; ++ j)
+    {
+      intermediateString = OSS() << " Coeff.#" << j;
+      oss << padString(intermediateString) << "|";
+    }
+    oss << "\n";
+  }
+  else
+  {
+      if (outputDimension == 1) 
+      {
+        intermediateString = OSS() << " Coefficient ";
+        oss << padString(intermediateString) << "|";
+        oss << "\n";
+      }
+      else
+      {
+        for (UnsignedInteger j = 0 ; j < outputDimension; ++ j)
+        {
+          intermediateString = OSS() << " Coeff.#" << j;
+          oss << padString(intermediateString) << "|";
+        }
+        oss << "\n";
+      }
+  }
+  // Print dashes
+  oss << "|-------|" << String(columnWidth, '-') << "|";
+  if (isEllipsisEnabled)
+  {
+    for (UnsignedInteger j = 0 ; j < 1 + 2 * ell_size; ++ j)
+      oss << String(columnWidth, '-') << "|";
+    oss << "\n";
+  }
+  else
+  {
+      if (outputDimension == 1) oss << String(columnWidth, '-') << "|\n";
+      else
+      {
+        for (UnsignedInteger j = 0 ; j < outputDimension; ++ j)
+          oss << String(columnWidth, '-') << "|";
+        oss << "\n";
+      }
+  }
+  // Print table content: multi-indices
+  for (UnsignedInteger i = 0; i < indicesSize; ++ i)
+  {
+    if (ellipsis && indicesSize > 2 * ell_size)
+    {
+      if (i == ell_size)
+      {
+        oss << "| ...   |      |             |";
+        for (UnsignedInteger j = 0 ; j < 1 + 2 * ell_size; ++ j)
+          oss << String(columnWidth, ' ') << "|";
```

Comment:
Create an intermediate string containing `String(columnWidth, ' ') << "|"` and repeat that string `1 + 2 * ell_size` times using the `String` constructor.

Reply:
Fixed at a3b84d52d0a1a8f8efd09354300f7de1af3f1a5e